concepts/decorators/about.md

```diff
@@ -1,2 +1,44 @@
-#TODO: Add about for this concept.
+# About
 
+Decorators are a feature of the Python language that can modify or register a function.

```

Comment:
Does "modify or register" narrow the definition a bit much? There's other ways a decorator might be used which I wouldn't put into either of those buckets.

Reply:
I'm not really sure what I could change this to?

Reply:
Could this work @IsaacG?
```suggestion
Decorators are a feature of the Python language that can affect what Python actually executes when a function is called.

```